### PR TITLE
Fix typo / grammar from rest-notebook.md

### PR DESCRIPTION
**Pull request by HeartSaVioR**
Just modifying doc so I don't file a new issue to JIRA.

### CODE REVIEW

File: docs/rest-api/rest-notebook.md [hunk 1]
```diff
@@ -75,8 +75,8 @@ limitations under the License.
     </tr>
     <tr>
       <td>Description</td>
-      <td>This ```POST``` method create a new notebook using the given name or default name if none given.
-          The body field of the returned JSON contain the new notebook id.
+      <td>This ```POST``` methods create a new notebook using the given name or default name if none given.
```
Comment (AhyoungRyu) [inline docs/rest-api/rest-notebook.md]: Hi @HeartSaVioR : ) 
I guess **This ```POST``` method creates** is correct, not **This ```POST``` methods create**.
Isn't it ? 
If I am right, plz check line number 132, 163.

Comment (HeartSaVioR) [inline docs/rest-api/rest-notebook.md]: @AhyoungRyu Yeah, right. My mistake.

**Comment by HeartSaVioR** [comment]
@AhyoungRyu Thanks for the review. It's ready for another round of review.

**Comment by Leemoonsoo** [comment]
Thanks @HeartSaVioR for the improvement.
Looks good to me!

**Comment by r-kamath** [comment]
LGTM :+1:

**Pull request closed without merging**